src/build/arm_quickstart/mainTemplate.json

```
@@ -0,0 +1,123 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "service_principal_id": {
+        "type": "string",
+        "metadata": {
+          "description": "A pre-generate service principal with contributor role attached for running the deployment."
+        }
+      },
+      "service_principal_secret": {
+        "type": "securestring",
+        "metadata": {
+          "description": "The sercret key for the service pincipal you created."
+        }
+      },
+      "subscription_id": {
+        "type": "string",
+        "defaultValue": "[subscription().subscriptionId]",
+        "metadata": {
+          "description": "Subscription ID for MissionLZ resources (defaults to context subscription)"
+        }
+      },
+      "mlz_location": {
+        "type": "string",
+        "defaultValue": "eastus",
+        "metadata": {
+          "description": "The location that you're deploying to, defaults to selected resource group location."
+        }
+      },
+      "tf_environment": {
+        "type": "string",
+        "defaultValue": "public",
+        "metadata": {
+          "description": "Terraform azurerm environment (defaults to 'public') see: https://www.terraform.io/docs/language/settings/backends/azurerm.html#environment"
+        }
+      },
+      "mlz_env_name": {
+        "type": "string",
+        "defaultValue": "[concat('mlz', parameters('utcValue'))]",
+        "metadata": {
+          "description": "The location that you're deploying to"
+        }
+      },
+      "no_bastion": {
+        "type": "string",
+        "defaultValue": "true",
+        "metadata": {
+          "description": "If any value is present, will not create Bastion Host and Jumpbox VM"
+        }
+      },
+      "no_sentinel": {
+        "type": "string",
+        "defaultValue": "true",
+        "metadata": {
+          "description": "If any value is present, will not create an Azure Sentinel solution"
+        }
+      },
```

```suggestion
      "no_bastion": {
        "type": "bool",
        "defaultValue": "true",
        "metadata": {
          "description": "If any value is present, will not create Bastion Host and Jumpbox VM"
        }
      },
      "no_sentinel": {
        "type": "bool",
        "defaultValue": "true",
        "metadata": {
          "description": "If any value is present, will not create an Azure Sentinel solution"
        }
      },
```

We could use the [bool type](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/data-types#booleans) and the UI will render a radio button-ish component.